controllers/provisioners/eks/helpers.go

```diff
@@ -379,7 +395,7 @@ func (ctx *EksInstanceGroupContext) GetBootstrapArgs() string {
 
 	labelsFlag := fmt.Sprintf("--node-labels=%v", strings.Join(ctx.GetLabelList(), ","))
 	taintsFlag := fmt.Sprintf("--register-with-taints=%v", strings.Join(ctx.GetTaintList(), ","))
-	return fmt.Sprintf("--kubelet-extra-args '%v %v %v'", labelsFlag, taintsFlag, bootstrapArgs)
+	return fmt.Sprintf("%v %v %v", labelsFlag, taintsFlag, bootstrapArgs)
```

Comment:
Are we changing existing behavior here?

Reply:
NVM, saw you wrapped it in 
```
func (ctx *EksInstanceGroupContext) GetBootstrapArgs() string {
	return fmt.Sprintf("--kubelet-extra-args '%v'", ctx.GetKubeletExtraArgs())
}
```

Reply:
Nope - this just seperates the building of the kubelet arg string from the placement of that into the bootstrap script.